src/main/java/nl/tudelft/contextproject/model/Inventory.java

```diff
@@ -3,13 +3,15 @@
 import java.util.ArrayList;
 
 import com.jme3.math.ColorRGBA;
+import nl.tudelft.contextproject.model.entities.Bomb;
+import nl.tudelft.contextproject.model.entities.Entity;
+import nl.tudelft.contextproject.model.entities.Key;
 
 /**
  * Class representing the players inventory.
- *
  */
 public class Inventory {
-	ArrayList<Entity> array; 
+	ArrayList<Entity> array;
```

Comment:
Perhaps this should be renamed to "pickedUpEntities" (or something like that) while we are at it.